sdk/loadtestservice/azure-developer-loadtesting/azure/developer/loadtesting/_polling.py

```diff
@@ -0,0 +1,217 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+import asyncio
+import logging
+import sys
+import time
+
+from azure.core.polling import PollingMethod, AsyncPollingMethod, LROPoller, AsyncLROPoller
+from typing import Any, Callable
+
+from azure.core.polling._poller import PollingReturnType
+
+if sys.version_info >= (3, 9):
+    from collections.abc import MutableMapping
+else:
+    from typing import MutableMapping  # type: ignore  # pylint: disable=ungrouped-imports
```

Comment:
Could you move the code into `_patch.py`? Otherwise it may be overrided or cleared by codegen.

Reply:
Done thanks for reminding me. I referred `_polling.py` from Azure Keyvault, just want to bring to notice that it might not be generated code. Changed pushed in latest commit.